### PR TITLE
add bugfix PRs to release notes

### PR DESCRIPTION
**Pull request by alisterburt**
# Description
bugfix PRs #3964 and #3963 added to release notes

cc @sofroniewn

### CODE REVIEW

**Review by kevinyamauchi — APPROVED** [review]
LGTM!

**Review by sofroniewn — CHANGES_REQUESTED** [review]
Can you put them at other end

File: docs/release/release_0_4_13.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ Complete list of changes below:
 
 
 ## Bug Fixes
+- Update plugin docs index (#3964)
```
Comment (sofroniewn) [inline docs/release/release_0_4_13.md]: We have them # in increasing order within a section - can these two get moved to end of section

**Review by sofroniewn — APPROVED** [review]
I just made order changes - no point holding things up

**Pull request merged**